### DISCUSSION TITLE
What to do next after claiming student developer pack? How to start creating project

### DISCUSSION THREAD

**Discussion by Prachi7249** [github/india#680] [Students]
I have completed all mendatory modules in self learning part and claimed GitHub students developer pack idk what to do next I'm completely new to git hub, how to make a project in git hub? My project need access to microsoft azure portel of AI? Please someone guide me

**Comment by Kamalesh3112** [comment]
You may need Microsoft Azure Cloud Subscription for using so I would suggest you to go for any alternative solutions
